example-configs/qubes-os-r4.2.yml

```
@@ -0,0 +1,154 @@
+git:
+  baseurl: https://github.com
+  prefix: QubesOS/qubes-
+  branch: master
+
+backend-vmm: xen
+debug: true
+verbose: true
+qubes-release: r4.2
+reuse-fetched-source: true
+
+distributions:
+  - host-fc32
+  - vm-bullseye
+  - vm-fc35
+
+templates:
+  - fedora-35:
+      dist: fc35
```

```suggestion
  - vm-fc36

templates:
  - fedora-36:
      dist: fc36
```

Generally, Fedora 36 should be used for new stuff, as it gets security fixes faster.